operator/v1alpha1/types_image_content_source_policy.go

```diff
@@ -47,9 +52,14 @@ type ImageContentSourcePolicyList struct {
 }
 
 // RepositoryDigestMirrors holds cluster-wide information about how to handle mirros in the registries config.
-// Note: the mirrors only work when pulling the images that are reference by their digests.
+// Note: the mirrors only work when pulling the images that are referenced by their digests.
 type RepositoryDigestMirrors struct {
-	// sources are repositories that are mirrors of each other.
+	// source is the repository that uses refer to, e.g. in image pull specifications.
+	// +required
+	Source string `json:"source"`
+	// mirrors is one or more repositories that may also contain the same images.
+	// Each mirror is tried in the order specified here* , and before the original “source”, which is contacted last.
```

Comment:
I don't want to enshrine the "contacted last" into the API guarantee.  You need to weaken this statement a bit, possibly:

```
The order of mirrors in this list is treated as the user's desired priority, while source is by default considered lower priority than all mirrors. Other cluster configuration may impact the exact order mirrors are contacted in, or some mirrors may be contacted in parallel, so this should be considered a preference rather than a guarantee of ordering.
```

Reply:
Updated to that text (and weaved in a note about conflicting `repositoryDigestMirrors`).